xbmc/pvr/windows/GUIEPGGridContainer.cpp

```diff
@@ -1567,55 +1567,49 @@ void CGUIEPGGridContainer::LoadLayout(TiXmlElement *layout)
   TiXmlElement *itemElement = layout->FirstChildElement("channellayout");
   while (itemElement)
   {
-    CGUIListItemLayout itemLayout;
-    itemLayout.LoadLayout(itemElement, GetParentID(), false, m_width, m_height);
-    m_channelLayouts.push_back(itemLayout);
+    m_channelLayouts.emplace_back();
```

Comment:
Seems something is missing here?

Reply:
I don't see that. It adds a new element, and invokes `LoadLayout`, just like the old code, only without the temporary. What is missing?

Reply:
You are right, it does call the default ctor; but there is still the `LoadLayout` call, identical to the old one, and it is needed.